perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_loa.java

```diff
@@ -0,0 +1,57 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.api.exceptions.*;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleImplApi;
+
+import java.util.List;
+
+/**
+ *
+ * @author Pavel Vyskocil vyskocilpavel@muni.cz
+ */
+public class urn_perun_user_attribute_def_virt_loa extends UserVirtualAttributesModuleAbstract  implements UserVirtualAttributesModuleImplApi {
+
+	public void checkAttributeValue(PerunSessionImpl sess, User user, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
+		if(!attribute.equals(getAttributeValue(sess, user, attribute))) throw new WrongAttributeValueException(attribute, user, "Attribute value is not the highest value from user's UserExtSources Loas.");
```

Comment:
This check doesn't make sense, since we never store attribute value and whenever we get it, it contains value. So e.g. for 100 users it would get their user ext. sources again to compare simple string/int. And if value has changed, is it really bad or not ? I guess only check on null should be present. What do you thing @licehammer?
If @licehammer will agree, please fix it also in ``member:virt:loa`` attribute module. Thank you.